sacc2/sacc.py

```diff
@@ -0,0 +1,1026 @@
+import numpy as np
+import copy
+import os
+import yaml
+import warnings
+import pickle
+
+from collections import OrderedDict
+from astropy.io import fits
+from astropy.table import Table, Column
+
+from .tracers import BaseTracer
+from .windows import BaseWindow
+from .covariance import BaseCovariance
+from .utils import unique_list
+
+
+# add more as we develop them
+allowed_types = [
+    "shear_xi_plus",
+    "shear_xi_minus",
+    "shear_ee",
```

Comment:
Also, we may want to store EB

Reply:
Will add!  I'm expecting that we'll gradually add more data types as time goes on; that's why the code doesn't cause an error if you use an unknown type, just a warning.

Reply:
I've added EB.  We'll need to make a fuller list of expected data types.